policyengine-client/src/countries/us/us.jsx

```diff
@@ -196,13 +211,18 @@ export class US extends Country {
                 "self_employment_income",
                 "dividend_income",
                 "interest_income",
+                "gi_cash_assistance",
```

Comment:
Calling guaranteed income / cash assistance `Market income` is a bit weird, but I think it fits here more than benefits, since it's not from the government.